src/scripts/main.js

```diff
@@ -1,3 +1,255 @@
 'use strict';
 
 // write code here
+const MIN_AGE = 18;
+const MAX_AGE = 90;
+const NOTIFICATION_TIMEOUT = 2000;
+
+const tableHeader = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const employeeForm = document.createElement('form');
+const submitButton = document.createElement('button');
+let lastSortedColumnIndex = null;
+let activeTableRow = null;
+
+employeeForm.classList.add('new-employee-form');
+
+const formFields = [
+  { label: 'Name', name: 'name', type: 'text' },
+  { label: 'Position', name: 'position', type: 'text' },
+  {
+    label: 'office',
```

Comment:
The label 'office' should be capitalized to maintain consistency with other field labels.